ibtk/include/ibtk/interpolation_utilities.h

```diff
@@ -0,0 +1,88 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2019 - 2021 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+#ifndef included_IBTK_interpolation_utilities
+#define included_IBTK_interpolation_utilities
+
+#include "ibtk/HierarchyGhostCellInterpolation.h"
+#include "ibtk/IBTK_MPI.h"
+#include "ibtk/IndexUtilities.h"
+
+#include "Box.h"
+#include "CartesianGridGeometry.h"
+#include "CartesianPatchGeometry.h"
+#include "CellIndex.h"
+#include "Index.h"
+#include "IntVector.h"
+#include "RobinBcCoefStrategy.h"
+#include "SAMRAI_config.h"
+#include "tbox/Pointer.h"
+
+#include <Patch.h>
+#include <PatchLevel.h>
+
+#include <vector>
+
+namespace IBTK
+{
+namespace Interpolation
+{
+/*
+ * Interpolations the cell centered data stored in patch index data_idx to the point X. Uses bilinear interpolation.
```

Comment:
Should we use `LEInteractor` under the hood and allow for user-specified interpolation kernels?

Reply:
We should definitely use `LEInteractor` - that would also correctly support multiple data centerings.